mycore-iview2/src/main/java/org/mycore/iview2/frontend/MCRThumbnailServlet.java

```diff
@@ -193,7 +193,7 @@ private BufferedImage getThumbnail(Path iviewFile, boolean centered) throws IOEx
             Path iviewFileRoot = fs.getRootDirectories().iterator().next();
             MCRTiledPictureProps props = MCRTiledPictureProps.getInstanceFromDirectory(iviewFileRoot);
             if (props.getZoomlevel() == 0) {
-                return MCRIView2Tools.getZoomLevel(iviewFile, 0);
+                return MCRIView2Tools.getZoomLevel(fs, 0);
```

Comment:
go with iviewFileRoot instead to match other methods.
Signature would not change but JavaDoc saying that Path has to be a directory (iviewFileRoot) if caller holds a ZipFileSystem open, if it is a file it is handled like a ZIP-File